### DISCUSSION TITLE
Add github sponsored region

### DISCUSSION THREAD

**Discussion by bourbonkk** [community/community#23618] [New to GitHub]
<p>When will Korea be added to the github sponsor regison?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>did you also request here?</p>
<p><a href="https://github.com/github/feedback/discussions" rel="noopener nofollow ugc">https://github.com/github/feedback/discussions</a></p>
<p>make sure when you create a new discussion, categorized it as Sponsors Feedback,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/c/dc3c9bd9bdb0bf8a78b50569f9228fe4494cd151.png" data-download-href="/uploads/short-url/vqiTCYf823iYXzVFcJkO3MLjwE9.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181078255-c2dbfb1a-c673-4456-89d0-a7b48fff3984.png" alt="image" data-base62-sha1="vqiTCYf823iYXzVFcJkO3MLjwE9" width="562" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dc3c9bd9bdb0bf8a78b50569f9228fe4494cd151_2_562x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dc3c9bd9bdb0bf8a78b50569f9228fe4494cd151_2_843x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/c/dc3c9bd9bdb0bf8a78b50569f9228fe4494cd151.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dc3c9bd9bdb0bf8a78b50569f9228fe4494cd151_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">987×877 52.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>I posted there and still waiting for the Philippines to be added too.</p>
